ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1559,3 +1559,60 @@ literals:
 
 ## New fields for seismic detector
 - seismic_alarm
+
+#New fields for DE-FRA-NEM32
+- lighting_color_cie_xvalue_sensor
```

Comment:
in this case, could we get rid of lighting color and just use cie xvalue sensor? or throw away cie and use `color_xvalue_sensor` it should be obvious it refers to light given its on a luminare/lighting controller/lighting gateway.

Reply:
fair enough, but then we would only get the tunable-white setpoint of the lighting group and not the feedback of the luminaire itself. But as mentioned above, in regards to other upcoming projects we need to clarify the tunable-white topic anyway.